indico/htdocs/js/indico/modules/designer.js

```diff
@@ -41,6 +41,16 @@
 
     var DEFAULT_PIXEL_CM = 50;
 
+    function _needs_param(field) {
+        if (field.type === "fixed") {
+            return true;
+        }
+        if (field.type === "genericfield"){
+            return true;
+        }
+        return false
```

Comment:
the whole function can be simplified to `return field.type == 'fixed' || field.type == 'genericfield';`